Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.17-alpine AS builder
+# Install git: required for fetching the Go dependencies.
+RUN apk update && apk add --no-cache git
+ENV GOPROXY=direct GOSUMDB=off
+WORKDIR /go/src/project
+ADD . .
+RUN env CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o vegatools .
+
+FROM alpine:3.14
```

Comment:
If we run a go binary a scratch image would do?

Reply:
It should as long as we don't require `ca-certificates` (i.e. we don't use rest endpoint).

Reply:
Do we? We can builtint the certificates in the binary if needed, I’ve done that often. (It’s not a big deal but it would r duce the size of image… probably just 10/15mb, but still smaller image = faster download etc.)